SwrveConversationSDK/Conversation/SwrveConversationItemViewController.m

```diff
@@ -63,6 +64,86 @@ - (void)viewDidDisappear:(BOOL)animated {
     }
 }
 
+- (void)dealloc {
```

Comment:
I've added this task as part of my refactor. https://swrvedev.jira.com/browse/SWRVE-13235